tomviz/ModuleRuler.cxx

```diff
@@ -150,8 +150,10 @@ bool ModuleRuler::setVisibility(bool val)
 {
   vtkSMPropertyHelper(m_Representation, "Visibility").Set(val ? 1 : 0);
   m_Representation->UpdateVTKObjects();
-  if (m_Widget) {
+  if (!val || m_showLine) {
```

Comment:
I am a fan of defensive programming, and would rather check for null even when you are fairly sure it won't happen. A comment would help a lot in this case.